endpoint-security-state/endpoint-security-state.yml

```diff
@@ -0,0 +1,240 @@
+        ##       when interval = "250ms", precision will be "1ms"
+        ## Precision will NOT be used for service inputs. It is up to each individual
+        ## service input to set the timestamp at the appropriate precision.
+        ## Valid time units are "ns", "us" (or "µs"), "ms", "s".
+        precision = ""
+
+        ## Logging configuration:
+        ## Run telegraf with debug log messages.
+        debug = false
+        ## Run telegraf in quiet mode (error log messages only).
+        quiet = false
+        ## Specify the log file name. The empty string means to log to stderr.
+        logfile = ""
+
+        ## Override default hostname, if empty use os.Hostname()
+        hostname = ""
+        ## If set to true, do no set the "host" tag in the telegraf agent.
+        omit_hostname = true
+        [[outputs.influxdb_v2]]	
+        ## The URLs of the InfluxDB cluster nodes.
+        ##
+        ## Multiple URLs can be specified for a single cluster, only ONE of the
+        ## urls will be written to each interval.
+        ## urls exp: http://127.0.0.1:9999
+        urls = ["$INFLUX_HOST"]
+        ## Token for authentication.
+        token = "$INFLUX_TOKEN"
+        ## Organization is the name of the organization you wish to write to; must exist.
+        organization = "$INFLUX_ORG"
+        ## Destination bucket to write into.
+        bucket = "telegraf"
+
+        ###############################################################################
+        #                            INPUT PLUGINS                                    #
+        ###############################################################################
+        #
+        # We must include the port for all of the URLs ( urls[] and sources[] ) below.
+        # x509_cert requires this and we need the URLs ( endpoints ) to match for
+        # display correlation
+        #
+
+        [[inputs.x509_cert]]
+          sources = [
+            "https://test.example.com:443/"
+          ]
+
+        [[inputs.http_response]]
+          urls = [
+            "https://test.example.com:443/"
+          ]
+
+        [[inputs.http_response]]
+          urls = [
+            "https://test.example.com:443/"
+          ]
+          username = "testuser"
+          password = "testpassword"
+
+        ###############################################################################
+        #                        PROCESSOR PLUGINS                                    #
+        ###############################################################################
+
+        [[processors.regex]]
+          [[processors.regex.tags]]
+            key = "source"
+            pattern = "^(.*)$"
+            replacement = "${1}"
+            result_key = "Endpoint"
+          [[processors.regex.tags]]
+            key = "server"
+            pattern = "^(.*)$"
+            replacement = "${1}"
+            result_key = "Endpoint"
```

Comment:
I think you're missing the bucket variable.

Reply:
Good catch, thank you!
It has now been added.